src/main/java/model/User.java

```diff
@@ -1,41 +1,66 @@
 package model;
 
+import java.util.ArrayList;
+
 public class User extends AbstractEntity {
     private String userName;
     private String password;
-    private boolean status;
+    private boolean isBlocked;
+    private UserRole role;
+    private ArrayList<String> order;
 
-    public void setStatus(boolean status) {
-        this.status = status;
+    public void setOrder(ArrayList<String> order) {
+        this.order = order;
     }
 
-    public void setPassword(String password) {
-        this.password = password;
+    public ArrayList<String> getOrder() {
+        return order;
     }
 
-    public void setUserName(String userName) {
-        this.userName = userName;
+    public boolean isBlocked() {
+        return isBlocked;
     }
 
-    public boolean isStatus() {
-        return status;
+    public void setBlocked(boolean blocked) {
+        isBlocked = blocked;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
     }
 
     public String getPassword() {
         return password;
     }
 
+    public void setUserName(String userName) {
+        this.userName = userName;
+    }
+
     public String getUserName() {
         return userName;
     }
 
+    public UserRole getRole() {
+        return role;
+    }
+
+    public void setRole(UserRole role) {
+        this.role = role;
+    }
+
+    enum UserRole {
```

Comment:
If you create an inner class(enum) - don't save it among getters and setters.  Save it in a separate class or higher(before getters and setters) better